docs/modules/tutorials/pages/web-app-to-rest-api.adoc

```diff
@@ -0,0 +1,64 @@
+= Web Application and REST API integration
+:noindex:
+
+:plugin-short-name: web-app-to-rest-api
+
+include::plugins:partial$plugin-installation.adoc[]
+
+== Steps
+include::plugins:partial$plugin-web-app-to-rest-api-steps.adoc[]
+
+== Table Transformers
+NOTE: For more information about Table Transforrmers, please refer to xref:ROOT:glossary.adoc#_table_transformer[Table Transformers]
```

Comment:
please follow the same format for headers across all docs (different ways to provide the same information may confuse users)